package.json

```diff
@@ -50,7 +50,7 @@
     "@gimenete/type-writer": "^0.1.3",
     "@gr2m/node-fetch": "^2.0.0",
     "@octokit/fixtures-server": "^3.0.0",
-    "@octokit/routes": "12.0.0",
+    "@octokit/routes": "14.0.0",
```

Comment:
Will this PR include the changes from https://github.com/octokit/routes/pull/267 based on this version bump?

Reply:
Yes :) Good point I’ll make sure to mention it in the release notes

Reply:
also I totally forgot that I still have to add the new endpoint to lib/routes.json as the file is not yet fully generated from octokit/routes, changes are only merged into it. But I’ve done that via 90aedbb now, so it’ll be in the next release. Thank you 🙏